dist/report/bitmaps_test/20230608-011702/report.json

```diff
@@ -0,0 +1,57 @@
+{
+  "testSuite": "BackstopJS",
+  "tests": [
+    {
+      "pair": {
+        "reference": "..\\bitmaps_reference\\Card_with_data-qa_card.png",
+        "test": "..\\bitmaps_test\\20230608-011702\\Card_with_data-qa_card.png",
+        "selector": "[data-qa=\"card\"]",
+        "fileName": "Card_with_data-qa_card.png",
+        "label": "Card with data-qa_card",
+        "requireSameDimensions": false,
+        "misMatchThreshold": 1,
+        "url": "http://localhost:3001/index.html",
+        "referenceUrl": "https://mate-academy.github.io/layout_solutions/product-cards/",
+        "expect": 0,
+        "viewportLabel": "tablet_h",
+        "diff": {
+          "isSameDimensions": false,
+          "dimensionDifference": {
+            "width": 2,
+            "height": 4
+          },
+          "misMatchPercentage": "37.81",
+          "analysisTime": 13
+        },
+        "diffImage": "..\\bitmaps_test\\20230608-011702\\failed_diff_Card_with_data-qa_card.png"
+      },
+      "status": "fail"
+    },
+    {
+      "pair": {
+        "reference": "..\\bitmaps_reference\\Link_with_data-qa_hover.png",
+        "test": "..\\bitmaps_test\\20230608-011702\\Link_with_data-qa_hover.png",
+        "selector": "[data-qa=\"card\"]",
```

Comment:
In the future - Do not add extra files to PR
To add the file to a commit use `git add src/.` & `git commit -m 'your message'`